diff --git a/config/psa_storage_user_config.h b/config/psa_storage_user_config.h
index 742f5d7..e5fc9b4 100644
--- a/config/psa_storage_user_config.h
+++ b/config/psa_storage_user_config.h
@@ -32,6 +32,6 @@
     #undef PSA_STORAGE_FILE_C_STORAGE_PREFIX
 #endif
 
-#define PSA_STORAGE_FILE_C_STORAGE_PREFIX "psa/"
+#define PSA_STORAGE_FILE_C_STORAGE_PREFIX "/userdata/mbed/mcc_config"
 
 #endif /* PSA_STORAGE_USER_CONFIG_H */
 